src/main/kotlin/lotto/PurchaseResult.kt

```diff
@@ -0,0 +1,26 @@
+package lotto
+
+import view.ResultView
+
+data class PurchaseResult(
+    val money: Money,
+    val tickets: List<Lotto>,
+    val manualCount: Int,
+) {
+    fun totalTicketCount(): Int = tickets.size
```

Comment:
`PurchaseResult` also seems to be a class that contains business logic,
but it's currently directly depending on the View. 😱 
It might be better to invert the dependency so that the `View` depends on `PurchaseResult` instead.